contrib/opentimelineio_contrib/adapters/advanced_authoring_format.py

```diff
@@ -1595,6 +1619,9 @@ def read_from_file(
     if simplify:
         result = _simplify(result)
 
+    # Make sure that any AAF Data tracks are marked as such
+    result = _update_kind_of_data_tracks(result)
+
```

Comment:
AAF's store markers on special slots called `EventSlots`.  The markers on these slots reference the slot(s) the are actually on by their `DescribedSlots` property.   The`_attach_markers` function already does this resolving, so this should not be needed.